package.json

```diff
@@ -21,6 +21,7 @@
   ],
   "main": "dist/server/honeybadger.js",
   "browser": "dist/browser/honeybadger.js",
+  "esm": "dist/esm/honeybadger.js",
```

Comment:
Is ESM the standard key? I usually use `"module"`, knowing npm, they're probably both right...

Reply:
> Is ESM the standard key? I usually use `"module"`, knowing npm, they're probably both right...

I'm not sure, tbh.

Reply:
Yeah probably it [should](https://esbuild.github.io/api/#main-fields) be `module`. I'm not sure why it's `esm` 🤔.

Reply:
> Yeah probably it [should](https://esbuild.github.io/api/#main-fields) be `module`. I'm not sure why it's `esm` 🤔.

Probably because I copied it from an example somewhere. 🤠

Reply:
Actually, I should take the blame on this one 😅 .
I cherry picked these changes from a fork of our repo (not available any more).